pkg/recipe/recipe.go

```diff
@@ -2,16 +2,13 @@ package recipe
 
 import (
 	"fmt"
-
-	"github.com/futurice/jalapeno/pkg/engine"
 )
 
 type Recipe struct {
 	Metadata  `yaml:",inline"`
 	Variables []Variable        `yaml:"vars,omitempty"`
 	Templates map[string][]byte `yaml:"-"`
 	Tests     []Test            `yaml:"-"`
-	engine    RenderEngine
```

Comment:
If the engine is no longer property of the recipe, I wonder how we can choose the right engine in the future if project uses multiple recipes with different engines

Reply:
I was thinking for example that Recipes specify the Engine with some sort of URI property, even maybe that "these files with this one, these with this other one"?

Reply:
I was thinking more simpler problem, where each recipe uses only one engine, but is different between two recipes used in a project

Reply:
Yeah, it would still work though in the simple case:
```yaml
apiVersion: v1
name: template-engine-test
version: v1.0.0
templateEngine: com.jalapeno.gotpl
templateExtension: .tmpl
...
```